packages/main/test/specs/Card.spec.js

```diff
@@ -124,6 +124,28 @@ describe("CardHeader", () => {
 		assert.strictEqual(await header2.getAttribute("aria-labelledby"), EXPECTED_ARIA_LABELLEDBY_HEADER2,
 			"The aria-labelledby is correctly set.");
 	});
+
+	it("test interactive card header accessibility role", async () => {
```

Comment:
Let's write these tests in Cypress, as we are going to nevertheless migrate all the wdio tests to Cypress.